.github/workflows/build.yaml

```diff
@@ -34,6 +35,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2.3.4
 
+      - name: start apt-cacher
+        run: docker buildx build --progress plain -t apt-cacher -f Dockerfile.apt-cacher . && docker run -d -p 3142:3142 apt-cacher
```

Comment:
Will the cache be reused across all jobs or only within-jobs?

Reply:
currently only for the current job. can be extended later to be more persistent